pkg/network/connection_test.go

```diff
@@ -0,0 +1,80 @@
+package network
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/alipay/sofa-mosn/pkg/log"
+	"github.com/alipay/sofa-mosn/pkg/types"
+)
+
+type MyEventListener struct{}
+
+func (el *MyEventListener) OnEvent(event types.ConnectionEvent) {}
+
+func randN() int {
+	return rand.Intn(1024) + 1
```

Comment:
global rand needs to set rand.Seed(time.Now().UnixNano())， or it will be use Seed as 0. 
At this test, can use a const N, no needs to use rand

Reply:
sorry, it should be inside fuzzy tests